src/applications/find-forms/api/index.js

```diff
@@ -44,5 +49,8 @@ export const fetchFormsApi = async (query, options = {}) => {
     });
   }
 
-  return sortBy(onlyValidForms, 'id');
+  return {
+    allFormsTombstone,
+    results: sortBy(onlyValidForms, 'id'),
```

Comment:
Is the `id` the same as the title of a form? 🙂 otherwise we may want to ask a PO what the desired sorting behavior should be!

Reply:
Yeah, that's default behavior for sorting. 
However, they're will be a knew sorting functionality coming soon!